test/substitutions_test.rb

```diff
@@ -1701,7 +1701,8 @@
 
       test 'should convert contents of asciimath macro to MathML in DocBook output if asciimath gem is available' do
         asciimath_available = !(Asciidoctor::Helpers.require_library 'asciimath', true, :ignore).nil?
-        input = 'asciimath:[a < b]'
+        # NOTE truffleruby fails if input equation contains spaces
+        input = 'asciimath:[a<b]'
```

Comment:
Again, is this something we should be fixing in TruffleRuby and Is there an associated issue?

Reply:
This is already been reported here https://github.com/asciidoctor/asciimath/issues/9 and here https://github.com/asciidoctor/asciimath/issues/10